app/gui/language/span-tree/src/lib.rs

```diff
@@ -77,19 +77,37 @@ use crate::generate::Context;
 pub struct ArgumentInfo {
     pub name:       Option<String>,
     pub tp:         Option<String>,
+    pub call_id:    Option<ast::Id>,
+    pub target_id:  Option<ast::Id>,
```

Comment:
what are these IDs? I know AST, but I dont understand what IDs are we referring here to.